src/main/java/Duke.java

```diff
@@ -1,10 +1,115 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
+        displayWelcomeMsg();
+
+        Task[] array = new Task[100];
+        String line;
+        Scanner in = new Scanner(System.in);
+        boolean looper = true;
+
+        while(looper) {
+            line = in.nextLine();
+            String lineCmd = line.trim().split(" ")[0];
+            switch(lineCmd){
+            case "bye":
+                displayByeMsg();
+                looper = false;
+                break;
+            case "list":
+                cmdList(array);
+                break;
+            case "done":
+                cmdDone(array, line);
+                break;
+            case "todo":
+                cmdTodo(array, line);
+                break;
+            case "deadline":
+                cmdDeadline(array, line);
+                break;
+            case "event":
+                cmdEvent(array, line);
+                break;
+            default:
+                System.out.println("Invalid command.");
+            }
+        } //end while loop
+    } //end of main
+
+    public static void cmdEvent(Task[] array, String line) {
+        int atPos = line.indexOf("/at");
+        String eventContent = line.trim().substring(6, atPos-1);
+        String eventAt = line.trim().split("/at ")[1];
+        Event newEvent = new Event(eventContent, eventAt);
+        array[Task.getNumTask()-1] = newEvent;
+        System.out.println("Got it. I've added this task:");
+        System.out.println("\t[E][\u2718] " + eventContent + " (at: " + eventAt + ")");
```

Comment:
Try use the toString method of event/ ddl/ toto?

Reply:
Thanks, i'll consider making this change next.